standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -577,6 +577,25 @@ public OpenTxnsResponse openTxns(OpenTxnRequest rqst) throws MetaException {
           LOG.debug("Going to execute update <" + q + ">");
           stmt.execute(q);
         }
+
+        if (rqst.isSetReplPolicy()) {
+          List<String> rowsRepl = new ArrayList<>();
+          String selectRepl = "select target_txn_id from TXN_MAP where repl_policy = " + quoteString(rqst.getReplPolicy()) + "and src_txn_id = " + rqst.getReplSrcTxnId();
+          for (long i = first; i < first + numTxns; i++) {
```

Comment:
This is misleading as one srcId cannot map to many txns. Either need to assert for numTxns as 1 here or use different method.